software/colanode.yml

```
@@ -0,0 +1,19 @@
+name: "Colanode"
+website_url: "https://colanode.com"
+source_code_url: "https://github.com/colanode/colanode"
+description: "Open-source and local-first Slack and Notion alternative that puts you in control of your data."
+licenses:
+  - Apache-2.0
+platforms:
+  - Javascript
+  - Nodejs
```

```suggestion
  - K8S
```
Docker and K8S seem to be the only distribution platform.